woocommerce/woocommerce
# [tests] Move API core tests as a suite in e2e-pw

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/49694

Core API and E2E tests have identical setup requirements, and both run using Playwright. To remove the maintenance burden this PR moves the API tests as tests in the e2e-pw solution. 

API and e2e are configured (in the default env) as playwright projects.

### How to test the changes in this Pull Request:

Locally, run the e2e and api tests, using the provided node scripts in package.json:
```
pnpm test:e2e
```
```
pnpm test:api
```

In CI, check that only and all the required tests are running in the e2e and api jobs:
- Core e2e tests
- Core API tests
